src/_companies/europe/germany/north_rhine-westphalia/easybill/index.md

```diff
@@ -0,0 +1,25 @@
+---
+name:  "easybill"
+
+continent: "Europe"
+country: "Germany"
+region: "North Rhine-Westphalia"
+city: "Willich"
+
+website: https://www.easybill.de/
+careers_page: https://www.easybill.de/jobs
+
+work_arrangements:
+  - remote
+  - hybdrid
+  - onsite
+
+stack:
+  - rails
+  - mysql
+
+created_at: 2023-03-21 09:32:23
+---
+
+easybill is a cloud-based invoicing software that has been successful on the market for more than 10 years. Currently we have more than 15,000 active customers and we are constantly growing. Therefore we are looking for motivated Ruby developers to join our team.
```

Comment:
Could you change this be written in the third person? Then it will be consistent with the rest of the company descriptions :)